packages/commerce-sdk-react/src/auth/storage.ts

```diff
@@ -6,10 +6,26 @@
  */
 import Cookies from 'js-cookie'
 
-export interface BaseStorage {
-    set(key: string, value: string, options?: unknown): void
-    get(key: string): string
-    delete(key: string): void
+export type StorageTypes = 'cookie' | 'local'
```

Comment:
```suggestion
export type StorageType = 'cookie' | 'local'
```
I feel like this kind of type is conventionally singular, but I could be wrong... 🤔

Reply:
I agree.